packages/code-du-travail-data/dataset/fiches_service_public/package.json

```diff
@@ -21,7 +21,6 @@
     "diff": "node diff.js",
     "updateTheme": "node updateTheme.js > out.json && mv out.json fiches-sp.json  && yarn report",
     "start": "yarn backup && node index.js > fiches-sp.json",
-    "report": "node diff.js && node missingTheme.js",
-    "test": "jest"
```

Comment:
They are no tests in this package !

Reply:
est ce que la regle s'applique partout ouil faut la rajouter par projet ?

Reply:
Elle s'applique à tout le mono repo, je l'ai mise à la racine